great_expectations/datasource/fluent/interfaces.py

```diff
@@ -409,7 +411,7 @@ def _execution_engine_type(self) -> Type[_ExecutionEngineT]:
 
     def get_execution_engine(self) -> _ExecutionEngineT:
         current_execution_engine_kwargs = self.dict(
-            exclude=self._EXCLUDED_EXEC_ENG_ARGS
+            exclude=self._EXCLUDED_EXEC_ENG_ARGS, config_provider=self._config_provider
```

Comment:
Example of using `.dict()` to resolve any config values for the execution engine. Like `connection_string` or other credentials.